ConsoleGame/ConsoleGameTests/UnitTest1.cs

```diff
@@ -0,0 +1,135 @@
+using ConsoleGame;
+
+namespace ConsoleGameTests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void CharacterStartXPositionTest()
+        {
+            Character character = new Character("====\n" +
+               "=@ =\n" +
+               "=  =\n" +
+               "====\n");
```

Comment:
```suggestion
            Character character = new Character(
               """
               ====
               =@ =
               =  =
               ====
               """);
```
Строки с переносами строк можно оформлять так. И нагляднее, и эффективнее.

Reply:
Одинаковую строку из всех тестов нужно вынести в константу.